src/TodoByRule.php

```diff
@@ -36,35 +52,58 @@ public function processNode(\PhpParser\Node $node, \PHPStan\Analyser\Scope $scop
             return [];
         }
 
-        $comments = $node->getComments();
-        if (count($comments) === 0) {
-            return [];
-        }
-
         $errors = [];
-        foreach($comments as $comment) {
-            $text = ltrim($comment->getText(), "\t /");
-            if (!str_starts_with($text, 'TODO')) {
```

Comment:
I used this check to prevent running the regex on every comment found, which is more efficient. running regex can be slow and memory inefficient depending on the comment text

Reply:
I did a few more tests locally on huge phpdoc comments and it seeems this is ok.